### ISSUE TITLE
In Google Chrome, NVDA reads modal titles three times when the modal is launched.

### ISSUE THREAD

**Issue by RD-Brown** [nvaccess/nvda#8221]
### Steps to reproduce:
Steps to Reproduce:
1.	In Chrome, open the attached file, HeaderReadingDemo.html.
2.	Press Enter over the Launch Modal button.
3.	Note that the modal title, “Test Submit Warning” is read three times.

### Expected behavior:
modal title, “Test Submit Warning” should only be read once.

### Actual behavior:
Modal title is read three times.

### System configuration:
NVDA version: 2018.1.1
NVDA Installed or portable: Installed
Windows version: Windows 10 Enterprise
Name and version of other software in use when reproducing the issue: Google Chrome V65.0.3325.181

### Other questions:
Does the issue still occur after restarting your PC? Yes.
Have you tried any other versions of NVDA? No.
[HeaderReadingDemo.zip](https://github.com/nvaccess/nvda/files/1964001/HeaderReadingDemo.zip)

**Comment by Adriani90** [comment]
@RD-Brown in Chrome 79 this example does not work for me. The modal does not launch when I press the button. Could you please give an update on this? In Firefox it seems NVDA repeats the text only one time.

**Comment by ObjectInSpace** [comment]
If you traverse the dialog with browse mode, NVDA reads two "test submit warning"s. So it says "test submit warning" when the dialog opens, then traverses the content from top to bottom including the two other  "test submit warning" strings. This behavior appears to be WAI for NVDA, and is identical in firefox. Curiously, JAWS only says test submit warning once. Not sure why. But I think that is either a problem in NVDA, or this reproduction example. A more useful reproduction example for this would have different strings for each instance of "test submit warning" (like "test submit warning 1/2/3") to more easily determine what exactly NVDA is reading and whether it should. But this does not appear to be a browser issue.

**Comment by Adriani90** [comment]
This is related to the setting automatic say all when page is loaded in the browse settings of NVDA. Turning that settip off does not move the focus automatically when entering a modal dialog. See also #8971.
